packages/sku/lib/configure.js

```diff
@@ -39,39 +39,44 @@ module.exports = async () => {
   ];
 
   // TODO: Remove this migration before releasing sku v15.
-  if (await shouldMigrateOldEslintConfig()) {
-    console.log("'.eslintignore' file detected. Attempting migration...");
+  const { shouldMigrate, eslintIgnoreExists } =
+    await shouldMigrateOldEslintConfig();
 
-    const customIgnores = migrateEslintignore({
-      hasLanguagesConfig: Boolean(languages && languages.length > 0),
-      target: paths.relativeTarget,
-    });
+  if (shouldMigrate) {
+    console.log("'.eslintignore' file detected. Attempting migration...");
```

Comment:
**suggestion:** We may want to change this message.

Or move it into the second if block. We may not have an ignorefile here.